jaypeasee/national-parkfinder
# Feature/router park info

## Is this a fix or a feature?
Feature

## What is the change?
1. Sets up Router for the entire app - `/:parkCode`, `/about` etc.
2. Gets a random park appear only when the route is `/`
3. Creates a controlled form on the MainNav

## Where should the reviewer start?
1. Review the return statement in App.
2. Review MainNav so see how our filter and controlled form now works.

## How should this be tested?
* Load the page, see that a random park loads when the path is `/`. 
* Click on a park to see the route change. Refresh to confirm you stay on that park and the route doesn't change
* Search for park buttons in the controlled form.

## Relevant Tickets
1. #34 
2. #22 
3. #23 

@saraho1123
@jaypeasee
@baileydunning
